README.md

```diff
@@ -1,68 +1,66 @@
-# Oracle Connector
+# Oracle Database Connector
 
 This is an Arrow connector to Oracle Database.
 
-## Installation
+## Prerequisites
+
+This connector requires Oracle Instant Client.
 
-You need to have the Oracle instant client installed on your computer in /opt/oracle/instantclient/
+To install it, follow the steps described here: https://github.com/oracle/node-oracledb/blob/master/INSTALL.md#instosx
```

Comment:
Please link to the Overview section and change the sentence to: To install them, please follow the instructions for your environment here: https://github.com/oracle/node-oracledb/blob/master/INSTALL.md#overview